physics/mp_thompson.meta

```diff
@@ -610,6 +610,14 @@
   type = real
   kind = kind_phys
   intent = out
+[max_hail_diam_sfc]
+  standard_name = max_hail_diameter_sfc
+  long_name = instantaneous maximum hail diameter at lowest model level
+  units = m
+  dimensions = (horizontal_loop_extent)
+  type = real
+  kind = kind_phys
+  intent = out
```

Comment:
This is defined with intent=inout in the .F90 file.
Please clarify.

Reply:
@dustinswales @joeolson42 to be consistent with other diagnostics like refl_10cm, this should be defined here as inout.